mobile/.gitignore

```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
This line is not necessary, or else we could add a consistent message across all the example project `.gitignore` files.